python/glow/gwas/lin_reg.py

```diff
@@ -114,16 +115,20 @@ def linear_regression(genotype_df: DataFrame,
     Q = np.linalg.qr(C)[0]
     Y = phenotype_df.to_numpy(dt, copy=True)
     Y_mask = (~np.isnan(Y)).astype(dt)
-    np.nan_to_num(Y, copy=False)
-    Y = gwas_fx._residualize_in_place(Y, Q)
+    Y = np.nan_to_num(Y, copy=False)
+    Y -= Y.mean(axis=0)  # Mean-center
+    Y = gwas_fx._residualize_in_place(Y, Q) * Y_mask  # Residualize
+    Y_scale = np.sqrt(np.sum(Y**2, axis=0) / (Y_mask.sum(axis=0) - Q.shape[1]))
+    Y /= Y_scale[np.newaxis, :]  # Scale
```

Comment:
nit: I think we typically use `None` instead of `np.newaxis`. They do the same thing.